bambou/session.go

```diff
@@ -84,32 +89,36 @@ func NewSession(username, password, organization, url string, root Rootable) *Se
 	}
 }
 
-// SetInsecureSkipVerify sets if the internal HTTP client should allow to connect
-// to insecure API endpoints.
-func (s *Session) SetInsecureSkipVerify(skip bool) *Error {
```

Comment:
What happens if we want to check the certificate? The customer might replace the certificates of Jboss with properly signed certificates, or certificates from an internal authority which is trusted across the internal network.

Code seems to automatically force insecure, skip verify, if i see the prepareHeaders method